src/algoliasearch.helper.js

```diff
@@ -346,12 +347,12 @@ AlgoliaSearchHelper.prototype.isRefined = function( facet, value ) {
 };
 
 /**
- * Check if the facet has any disjunctive or conjunctive refinements
- * @param {string} facet the facet attribute name
- * @return {boolean} true if the facet is facetted by at least one value
+ * Check if the attribute has any numeric, disjunctive or conjunctive refinements
+ * @param {string} attribute the name of the attribute
+ * @return {boolean} true if the attribute is filtered by at least one value
  */
-AlgoliaSearchHelper.prototype.hasRefinements = function( facet ) {
-  return this.isRefined( facet );
+AlgoliaSearchHelper.prototype.hasRefinements = function( attribute ) {
+  return !isEmpty( this.state.getNumericRefinements( attribute ) ) || this.isRefined( attribute );
```

Comment:
From an end-user POV, I don't really get the difference between `isRefined` and `hasRefinements`. Is `isRefined` a lower level method?

If you had to split the two boolean checks (`!isEmpty( this.state.getNumericRefinements( attribute ) )` and `this.isRefined( attribute )` in variables, how would you call them? This could make the code easier to grasp.